packages/lib/src/interactions/RatioSelectionRect.tsx

```diff
@@ -15,22 +15,19 @@ function RatioSelectionRect(props: Props) {
   const { startPoint, endPoint, ratio, ...svgProps } = props;
   const { visSize, dataToWorld, worldToData } = useVisCanvasContext();
 
-  const [dataStartPoint, dataEndPoint] = getRatioRespectingRectangle(
-    startPoint,
-    endPoint,
-    ratio
-  ).map(dataToWorld);
+  const [ratioWorldStartPoint, ratioWorldEndPoint] =
+    getRatioRespectingRectangle(startPoint, endPoint, ratio).map(dataToWorld);
 
-  const [worldStartPoint, worldEndPoint] = clampRectangleToVis(
-    dataStartPoint,
-    dataEndPoint,
+  const [clampedStartPoint, clampedEndPoint] = clampRectangleToVis(
```

Comment:
```suggestion
  const [clampedDataStartPoint, clampedDataEndPoint] = clampRectangleToVis(
```
following your previous namings ?